### DISCUSSION TITLE
418CA3CC-FC93-4F5B-B499-BB3CC649C0EF

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#17] [General]
# HD 10180

HD 10180 is a Sun-like star about 127.22 light-years away in the constellation Hydrus. It is notable for having the same number of planets as our own Solar System, but the existence of only six of them are confirmed, the other two are still planet candidates as of 2022. The tally previously was as many as 9, but follow up studies for the innermost candidate (planet b) turns out to be a false positive
NOTE: This addon only includes textures for the six confirmed planets in this system. The remaining three will be uploaded later in two separate addons

https://celestia.mobi/addon?item=418CA3CC-FC93-4F5B-B499-BB3CC649C0EF&lang=en

**Comment by exocritic5457** [comment]
are the 'remaining three' ever going to be added?

Also d, g, and h have seams on their textures

**Reply by exocritic5457** [reply]
It's been a month... any details would be nice

**Reply by SevenSpheres** [reply]
The three candidates are included in the original addon, they're just waiting for someone to submit them to mobi. [You can even do this yourself.](https://celestia.mobi/submit-addon)
